pyaedt/generic/touchstone_parser.py

```diff
@@ -475,3 +477,74 @@ def read_touchstone(file_path):
     """
     data = TouchstoneData(touchstone_file=file_path)
     return data
+
+
+@pyaedt_function_handler()
+def check_touchstone_files(folder="", passivity=True, causality=True):
+    """Check passivity and causality for all touchstone files included in folder.
+
+    Parameters
+    ----------
+    folder : str
+        folder path.
+    passivity : bool, optional
+        Whether the passivity check is enabled, The default is ``True``.
+    causality : bool, optional
+        Whether the causality check is enabled. The default is ``True``.
+    Returns : dict
+        Dictionary with snp file name as key, list for passivity and causality is enabled. First element from list
+        is str with ``"passivity"`` or ``"causality"`` as value. Second element bool with ``True`` when criteria passed
+        ``False`` otherwise. Last element str with the log information.
```

Comment:
```suggestion
        Dictionary with the SNP file name as the key and a list if the passivity and/or causality checks are enabled.
        The first element in the list is a str with ``"passivity"`` or ``"causality"`` as a value. The second element
        is a Boolean that is set to ``True`` when the criteria passed or ``False`` otherwise. The last element
        is a string with the log information.
```
It seems that both the passivity and causality checks can be enabled as there is no indication in the parameter descriptions that it can only be one or the other. Consequently, I used "and/or" in the first line of the Returns description. However, the description for the first line seems to imply it is one or another. Please clarify.